evap/evaluation/migrations/0085_auto_20181015_1908.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.0.9 on 2018-10-15 19:08
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+def delete_cronjob_user(apps, _schema_editor):
+    UserProfile = apps.get_model('evaluation', 'UserProfile')
+    try:
+        UserProfile.objects.get(username='cronjob').delete()
+    except UserProfile.DoesNotExist:
+        pass
```

Comment:
i would expect the user to always exist, so no need for the error handling.

please add a reverse part to the migration (for examples, search for `reverse_code` in the migrations). that is, when the migration is undone (e.g. in order to go back to an older revision to debug things), the cronjob user would need to be created again.

Reply:
makes sense... i added a reverse_code function

Reply:
just saw this - we can either add the few lines again to prevent the migration fail on testing systems or (as it is not important for production) we could argue that testing instances need to be reset after applying this pr.